content/en/continuous_integration/tests/javascript.md

```diff
@@ -41,7 +41,7 @@ Supported test frameworks:
 The instrumentation works at runtime, so any transpilers such as TypeScript, Webpack, Babel, or others are supported out of the box.
 
 ### Test suite level visibility compatibility
-[Test suite level visibility][5] is fully supported from `dd-trace>=3.14.0` and `dd-trace>=2.27.0`. Jest, Mocha, Playwright, Cypress, and Cucumber are supported.
+[Test suite level visibility][5] is fully supported from `dd-trace>=3.14.0` and `dd-trace>=2.27.0`. Jest, Mocha, Playwright, Cypress and Cucumber are supported.
```

Comment:
```suggestion
[Test suite level visibility][5] is fully supported from `dd-trace>=3.14.0` and `dd-trace>=2.27.0`. Jest, Mocha, Playwright, Cypress, and Cucumber are supported.
```

Reply:
Can we leave the comma here? It's actually part of our style guide to use the Oxford comma, so for consistency we prefer to keep it.

Reply:
oh sure, I didn't know that. Thanks